File: game-admin/src/main/java/com/dkm/feign/fallback/AnotherUserFeignClientFallback.java
package com.dkm.feign.fallback;

import com.dkm.admin.entities.vo.UserLoginVo;
import com.dkm.constanct.CodeType;
import com.dkm.data.Result;
import com.dkm.feign.AnotherUserFeignClient;
import com.dkm.feign.entity.UserLoginStatusVo;
import kotlin.Metadata;
import kotlin.jvm.internal.Intrinsics;
import org.jetbrains.annotations.NotNull;
import org.springframework.stereotype.Component;

@Component
public class AnotherUserFeignClientFallback implements AnotherUserFeignClient {
   @NotNull
   public Result<UserLoginStatusVo> login(@NotNull UserLoginVo userLoginVo) {
      return Result.fail(CodeType.FEIGN_CONNECT_ERROR);
   }
}

File: game-admin/src/main/java/com/dkm/feign/fallback/AnotherUserFeignClientFallback.kt
package com.dkm.feign.fallback

import com.dkm.admin.entities.vo.UserLoginVo
import com.dkm.constanct.CodeType
import com.dkm.data.Result
import com.dkm.feign.AnotherUserFeignClient
import com.dkm.feign.entity.UserLoginStatusVo
import org.springframework.stereotype.Component


@Component
class AnotherUserFeignClientFallback :AnotherUserFeignClient  {

    override fun login(userLoginVo: UserLoginVo): Result<UserLoginStatusVo> {
        return Result.fail<UserLoginStatusVo>(CodeType.FEIGN_CONNECT_ERROR)
    }

}
